perun-core/src/main/java/cz/metacentrum/perun/core/impl/TasksManagerImpl.java

```diff
@@ -125,40 +125,25 @@ public int insertNewTaskResult(TaskResult taskResult, int engineID) throws Inter
 				"insert into tasks_results(" +
 					"id, " +
 					"task_id, " +
+					"engine_id, " +
 					"destination_id, " +
 					"status, " +
 					"err_message, " +
 					"std_message, " +
 					"return_code, " +
-					"timestamp, " +
-					"engine_id) values (?,?,?,?,?,?,?, " + Compatibility.toDate("?","'DD-MM-YYYY HH24:MI:SS'") + " ,?)",
+					"timestamp) values (?,?,?,?,?,?,?, " + Compatibility.toDate("?","'DD-MM-YYYY HH24:MI:SS'") + ")",
 				newTaskResultId,
 				taskResult.getTaskId(),
+				0,
```

Comment:
We must default to ``1`` with ``engine_id``, since we are not changing DB schema yet and we have constraint on existing engine entry with ID=1 on each instance.

Reply:
Fixed.